src/core/catalog.js

```diff
@@ -130,6 +130,15 @@ class Catalog {
     return shadow(this, "acroForm", acroForm);
   }
 
+  get acroFormRef() {
+    let acroFormRef = null;
+    const value = this._catDict.getRaw("AcroForm");
+    if (isRef(value)) {
+      acroFormRef = value;
+    }
+    return shadow(this, "acroFormRef", acroFormRef);
+  }
```

Comment:
This could be simplified slightly; how about just:
```javascript
const value = this._catDict.getRaw("AcroForm");
return shadow(this, "acroFormRef", isRef(value) ? value : null);
```